tests/tests/recovery.rs

```diff
@@ -167,6 +167,8 @@ async fn _gas_price_updater__can_recover_on_startup_when_gas_price_db_is_behind(
         Some(BlockHeight::new(height))
     );
 
+    tokio::time::sleep(std::time::Duration::from_millis(20)).await;
```

Comment:
Do you want to sneak this change or it's leftover ?

Reply:
No, on my machine, this test was failing after the `run` function became non blocking